internal/config/config.go

```diff
@@ -1,8 +1,6 @@
-package internal
+package config
```

Comment:
It's a bad decision. A package should be independent reusable entity, config is just part of wal-g.

Reply:
I just didn't know how else I could resolve circular imports. crypto/openpgp needs config functions from internal, internal needs crypto/openpgp. I guess I should somehow move those config functions out of crypto package to make it an independent reusable entity and then erase this internal/config package.

Reply:
Undone this.